Compilation fails to find MIN definition

```
...
[4/41] Compiling C object 'subprojects.../bbec2f1@@neatvnc@sha/src_pixels.c.o'.
../subprojects/neatvnc/src/pixels.c: In function ‘pixel32_to_cpixel’:
../subprojects/neatvnc/src/pixels.c:52:11: warning: unused variable ‘dst_blue_max’ [-Wunused-variable]
  uint32_t dst_blue_max = dst_fmt->blue_max;
           ^~~~~~~~~~~~
../subprojects/neatvnc/src/pixels.c:51:11: warning: unused variable ‘dst_green_max’ [-Wunused-variable]
  uint32_t dst_green_max = dst_fmt->green_max;
           ^~~~~~~~~~~~~
../subprojects/neatvnc/src/pixels.c:50:11: warning: unused variable ‘dst_red_max’ [-Wunused-variable]
  uint32_t dst_red_max = dst_fmt->red_max;
           ^~~~~~~~~~~
[5/41] Compiling C object 'subprojects...nc/bbec2f1@@neatvnc@sha/src_zrle.c.o'.
../subprojects/neatvnc/src/zrle.c: In function ‘zrle_encode_box’:
../subprojects/neatvnc/src/zrle.c:207:6: warning: unused variable ‘chunk_size’ [-Wunused-variable]
  int chunk_size = 1 + bytes_per_cpixel * TILE_LENGTH * TILE_LENGTH;
      ^~~~~~~~~~
[37/41] Compiling C object 'wayvnc@exe/src_keyboard.c.o'.
../src/keyboard.c: In function ‘keyboard_init’:
../src/keyboard.c:169:2: warning: ignoring return value of ‘write’, declared with attribute warn_unused_result [-Wunused-result]
  write(keymap_fd, keymap_string, keymap_len);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[38/41] Compiling C object 'subproject.../bbec2f1@@neatvnc@sha/src_server.c.o'.
../subprojects/neatvnc/src/server.c: In function ‘on_client_set_encodings’:
../subprojects/neatvnc/src/server.c:311:20: warning: implicit declaration of function ‘MIN’; did you mean ‘CMIN’? [-Wimplicit-function-declaration]
  int n_encodings = MIN(MAX_ENCODINGS, ntohs(msg->n_encodings));
                    ^~~
                    CMIN
../subprojects/neatvnc/src/server.c: In function ‘on_client_update_fb_done’:
../subprojects/neatvnc/src/server.c:718:15: warning: unused variable ‘server’ [-Wunused-variable]
  struct nvnc* server = client->server;
               ^~~~~~
At top level:
../subprojects/neatvnc/src/server.c:63:20: warning: ‘fourcc_to_string’ defined but not used [-Wunused-function]
 static const char* fourcc_to_string(uint32_t fourcc)
                    ^~~~~~~~~~~~~~~~
[39/41] Linking target subprojects/neatvnc/libneatvnc.so.0.0.0.
FAILED: subprojects/neatvnc/libneatvnc.so.0.0.0 
cc  -o subprojects/neatvnc/libneatvnc.so.0.0.0 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_server.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_util.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_vec.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_zrle.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_raw-encoding.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_pixels.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_damage.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_fb.c.o' 'subprojects/neatvnc/bbec2f1@@neatvnc@sha/contrib_miniz_miniz.c.o' -Wl,--as-needed -Wl,--no-undefined -Wl,-O1 -shared -fPIC -Wl,--start-group -Wl,-soname,libneatvnc.so.0 -lm /usr/lib/x86_64-linux-gnu/libpixman-1.so /usr/lib/x86_64-linux-gnu/libuv.so -lrt -lpthread /usr/lib/x86_64-linux-gnu/libnsl.so -ldl -Wl,--end-group
subprojects/neatvnc/bbec2f1@@neatvnc@sha/src_server.c.o: In function `on_client_read':
server.c:(.text+0xd45): undefined reference to `MIN'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

--- a/src/server.c
+++ b/src/server.c
@@ -30,6 +30,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/queue.h>
+#include <sys/param.h>
 #include <assert.h>
 #include <uv.h>
 #include <libdrm/drm_fourcc.h>

